src/core/structures/PDFAcroFormField.ts

```diff
@@ -0,0 +1,75 @@
+import {
+  PDFArray,
+  PDFDict,
+  PDFName,
+  PDFNumber,
+  PDFObject,
+  PDFString,
+} from 'src/core';
+
+const acroFormFieldTypes: Array<PDFObject | undefined> = [
+  PDFName.Btn,
+  PDFName.Ch,
+  PDFName.Tx,
+  PDFName.Sig,
+];
+
+class PDFAcroFormField {
```

Comment:
What do you think about shortening this class name to `PDFAcroField`? I think it's just as clear, and the brevity makes it easier to read/use.

Reply:
Updated